plugins/package-managers/spm/src/main/kotlin/SpmModel.kt

```diff
@@ -136,35 +131,14 @@ data class AppDependency(
 
     override val id: Identifier
         get() {
-            val (author, _) = parseAuthorAndProjectFromRepo(repositoryUrl)
             return Identifier(
                 type = PACKAGE_TYPE,
-                namespace = author.orEmpty(),
-                name = packageName,
+                namespace = "",
+                name = getPackageName(repositoryUrl),
                 version = state?.toString().orEmpty()
             )
         }
 }
 
-private val logger = loggerOf(MethodHandles.lookup().lookupClass())
-
-internal fun parseAuthorAndProjectFromRepo(repositoryURL: String): Pair<String?, String?> {
-    val normalizedURL = normalizeVcsUrl(repositoryURL)
-    val vcsHost = VcsHost.fromUrl(URI(normalizedURL))
-    val project = vcsHost?.getProject(normalizedURL)
-    val author = vcsHost?.getUserOrOrganization(normalizedURL)
-
-    if (author.isNullOrBlank()) {
-        logger.warn {
-            "Unable to parse the author from VCS URL $repositoryURL, results might be incomplete."
-        }
-    }
-
-    if (project.isNullOrBlank()) {
-        logger.warn {
-            "Unable to parse the project from VCS URL $repositoryURL, results might be incomplete."
-        }
-    }
-
-    return author to project
-}
+private fun getPackageName(repositoryUrl: String): String =
+    repositoryUrl.replaceCredentialsInUri().removeSchemeFromUri().removeSuffix(".git")
```

Comment:
I propose to replace `removeSchemeFromUri()` with simply `substringAfter("://")`.